data/sql/updates/pending_db_world/rev_1548473216184905600.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1548473216184905600');
+
+UPDATE `creature_template` SET `ScriptName`='', `AIName`='SmartAI' WHERE `entry`=1855;
+
+DELETE FROM `smart_scripts` WHERE `entryorguid`=1855 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`event_param5`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_param4`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(1855,0,0,1,62,0,100,0,3683,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,0,""),
```

Comment:
I'll just give you **Example** : 
(1855,0,0,1,62,0,100,0,3683,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,0,"**Show Gossip Menu for 3683**"),

Fill ''comment'' row with text description